awx/api/views/__init__.py

```diff
@@ -204,20 +204,14 @@ def get(self, request, format=None):
                                             'failed': ec2_inventory_failed.count()}
 
         user_groups = get_user_queryset(request.user, models.Group)
-        groups_job_failed = (
```

Comment:
This makes group failure tracking no longer exist/supported at `/api/v2/dashboard/`.

Reply:
It feels like this was for the old inventory model when groups and inventory sources were tied together.

Reply:
Yep, that's my impression, too.

Reply:
👍